plugin/src/main/java/org/wildfly/plugin/provision/PackageServerMojo.java

```diff
@@ -174,14 +177,81 @@ public class PackageServerMojo extends AbstractProvisionServerMojo {
     @Parameter(defaultValue = "false", property = PropertyNames.SKIP_PACKAGE_DEPLOYMENT)
     protected boolean skipDeployment;
 
+    /**
+     * Galleon provisioning info discovery. By enabling this feature, the set of Galleon feature-packs
+     * and layers are automatically discovered by scanning the deployed application.
+     * You can configure the following items:
```

Comment:
Minor again, but we should probably surround this in a `<p></p>`. It will just format better for the documentation.

Reply:
Ok (I hope what I did is what you suggested).